File: Arbitration/MPArbitration/ClientApp/src/app/model/case-log.ts
export class CaseLog {
    public arbitrationCaseId = 0;
    public id = 0;
    public action = '';
    public details = '';
    public createdBy = '';
    public createdOn: Date|undefined;

    constructor(obj?:any) {
        if(!obj)
            return;
        Object.assign(this, JSON.parse(JSON.stringify(obj)));
        if(this.createdOn)
            this.createdOn = new Date(obj.createdOn);
    }
}

File: Arbitration/MPArbitration/Model/CaseLog.cs
using System.Text.Json.Serialization;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations;

namespace MPArbitration.Model
{
    public class CaseLog
    {
        [JsonPropertyName("id")]
        public int Id { get; set; } = 0;
        [StringLength(20)]
        [JsonPropertyName("action")]
        public string Action { get; set; } = "";
        [JsonPropertyName("details")]
        public string Details { get; set; } = "";
        [JsonPropertyName("createdBy")]
        [StringLength(50)]
        public string CreatedBy { get; set; } = "";
        [JsonPropertyName("createdOn")]
        public DateTime? CreatedOn { get; set; } = null;

        [JsonPropertyName("arbitrationCaseId")]
        public int ArbitrationCaseId { get; set; } = 0;
    }
}
